nixos/modules/services/misc/home-assistant.nix

```diff
@@ -225,7 +228,7 @@ in {
     systemd.services.home-assistant = {
       description = "Home Assistant";
       after = [ "network.target" ];
-      preStart = optionalString (cfg.config != null) (if cfg.configWritable then ''
```

Comment:
So the actual bug fix is this line?

Reply:
Yeah but I believe `frontend` missing was also causing issues. It's been quite a while though and it's not that fresh anymore. I'll have to spend some time going through and running it in various situations.